nixos/modules/misc/version.nix

```diff
@@ -75,7 +75,7 @@ in
       };
 
       release = mkOption {
-        readOnly = true;
+        internal = true;
```

Comment:
I'm very much against setting those options as not internal. Downstreams know where to look, and there's no reason to have users be exposed to internal distro branding options.

Reply:
> Downstreams know where to look and can override them,

Am I misunderstanding something? `internal` omits the option from the manual while `readOnly` options are also *cannot* be overriden. Or is your point that such downstream use-cases must/ought fork nixpkgs and manually update the field? In any case, I agree is #351736 is a much better solution.